### ISSUE TITLE
HTML encoded strings

### ISSUE THREAD

**Issue by goblindegook** [marisaroque/sketch-markup-listify#3]
Exported strings aren't HTML encoded. For example:

`<li>Ampersands & Co. <3</li>`

Should be rendered as:

`<li>Ampersands &amp; Co. &lt;3</li>`

**Comment by marisaroque** [comment]
Yes, you are totally right, @goblindegook. I will fix it right away. 
Thank you very much for letting me know!

**Comment by marisaroque** [comment]
@goblindegook, I committed this change [https://github.com/marisaroque/sketch-markup-listify/commit/9917f081ed84d6db0025ae4b8608e1c63171123a]. 

I think it's fixed now. I'll close the issue. 

Thank you again!

**Comment by goblindegook** [comment]
Looks good! I made a couple of suggestions, basically just stuff you can remove from your code without worry.

Thanks!
